lib/static.js

```
@@ -15,6 +15,10 @@ var serialize = require('dom-serializer'),
  */
 
 exports.load = function(content, options) {
+  if (content === null || content === undefined) {
+    throw new TypeError('cheerio.load() expects a string');
```

Please throw a plain error.